src/platform/haiku/haiku.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PLATFORM_HAIKU_H
+#define PLATFORM_HAIKU_H
+
+#ifdef __HAIKU__
+
+#define PLATFORM_USE_SOFTWARE_CURSOR
```

Comment:
Just out of curiosity, don't the regular cursors work? Using the software cursor has some drawbacks as the cursor position will only update according to the game's fps.

Reply:
They did not for me. Without software cursor set to true, I had an invisible cursor in both VMware and on a laptop. Mousing over menu items did indicate that the mouse was working, but you couldn't see it unless there was a mouseover change, like a highlight on a button.

Doing a bit of research, I found references to an SDL bug in the Haiku implementation, and indications that hardware cursors weren't fully implemented. So it's most likely one of those two things forcing this.

Reply:
Do you have any links for the fact that hardware cursors aren't fully implemented?
Which version of SDL does Haiku use? I saw a [commit](https://github.com/libsdl-org/SDL/commit/72bcf546f90423f05eda0fa08510e340683ff6f4) on SDL for Haiku cursor code. Does it make sense to wait for SDL 2.0.16 - or at least re-check this later?